ports/esp32/mpconfigport.h

```diff
@@ -260,6 +260,20 @@ typedef long mp_off_t;
 // board specifics
 #define MICROPY_PY_SYS_PLATFORM "esp32"
 
+// Enable stdio over native USB peripheral CDC via TinyUSB
+#ifndef MICROPY_HW_USB_CDC
+#define MICROPY_HW_USB_CDC (SOC_USB_OTG_SUPPORTED)
+#endif
+
+// Enable stdio over USB Serial/JTAG peripheral
+#ifndef MICROPY_HW_ESP_USB_SERIAL_JTAG
+#define MICROPY_HW_ESP_USB_SERIAL_JTAG (SOC_USB_SERIAL_JTAG_SUPPORTED && !MICROPY_HW_USB_CDC)
+#endif // MICROPY_HW_ESP_USB_SERIAL_JTAG
```

Comment:
The comment on this line is not needed, because the if-endif is so short, and it's not consistent with the other if-endif blocks in this file.